test/integration/utils.js

```diff
@@ -0,0 +1,53 @@
+import path from 'path';
+
+import { BrowserWindow } from 'electron';
+
+const UI_DIR = path.join(__dirname, '..', '..', 'lib', 'ui');
+const BROWSER_CHROME_URL = fileUrl(path.join(UI_DIR, 'browser-modern', 'browser.html'));
+
+function fileUrl(str) {
+  let pathName = path.resolve(str).replace(/\\/g, '/');
```

Comment:
So this gets to the root of a problem with this approach. I can import app/shared/paths-utils here but it will be a different instance of the module to the one in the webpack bundle. In this case that doesn't matter, the module doesn't store any state but I worry that for other cases it will and we'll get confused when we expect them to be the same instance.